PowerShellTools/DebugEngine/ScriptDebugger.cs

```diff
@@ -464,7 +468,16 @@ internal void OpenRemoteFile(string fullName)
                 }
                 catch (Exception ex)
                 {
-                    Log.Error("Failed to open remote file through powershell remote session", ex);
+                    String err;
+                    if (forRemote)

```

Comment:
See above for decision about using flags in this method: https://github.com/Microsoft/poshtools/pull/620#discussion-diff-33091976. 

But I agree, with general statement that flags are no good. Continue that discussion here: https://github.com/Microsoft/poshtools/pull/620#discussion-diff-33091081